src/include/catalog/column_catalog.h

```diff
@@ -86,7 +88,8 @@ class ColumnCatalog : public AbstractCatalog {
                     oid_t column_id, oid_t column_offset,
                     type::TypeId column_type, bool is_inlined,
                     const std::vector<Constraint> &constraints,
-                    type::AbstractPool *pool, concurrency::TransactionContext *txn);
+                    type::AbstractPool *pool, concurrency::TransactionContext *txn,
+                    type::TypeId column_elem_type = type::TypeId::INVALID);
```

Comment:
Rearrange parameters to pool and txn come at the end to align with other catalog object constructors.